src/welcome/getting-started.js

```diff
@@ -0,0 +1,164 @@
+			link: '#',
+			icon: <SVGTutorialsIcon />,
+		},
+		{
+			title: __( 'Documentation and Support', i18n ),
+			subtitle: __( 'Get assistance with troubleshooting, guides, FAQs, and updates.', i18n ),
+			link: '#',
+			icon: <SVGDocsIcon />,
+		},
+		{
+			title: __( 'Community', i18n ),
+			subtitle: __( 'Join our very active Stackable Community on Facebook.', i18n ),
+			link: '#',
+			icon: <SVGCommunityIcon />,
+		},
+	],
+}
+
+const essentialsProps = {
+	label: 'essentials',
+	content: [
+		{
+			title: __( 'How to Add Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Customize Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use Copy & Paste Styles', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'Introduction to Wireframes', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use the Design Library', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use the Column Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+	],
+	headerTitle: __( 'Learn the essentials', i18n ),
+	headerSubtitle: __( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Consequat tempus sollicitudin rhoncus pretium tellus.', i18n ),
+}
+
+const Section = props => {
+	const Card = ( {
+		title, subtitle, link, icon = '', src = '',
+	} ) => {
+		return <div className="s-card">
+			{ props.label === 'general'
+				? <div className="s-icon-wrapper"> { icon } </div>
+				: <div className="s-video-wrapper">
+					<iframe className="s-video" src={ src } title={ title } allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture;" allowFullScreen></iframe>
+				</div>
+			}
+			<h3 className="s-card-title"> { title } </h3>
+			<p className="s-card-subtitle"> { subtitle } </p>
+			<div className="s-card-link">
+				<a href={ link }>
+					{ props.label === 'general' ? <SVGArrowUpRightIcon />
+						: __( 'Watch Video', i18n )
```

Comment:
For the video component, please remove the "Watch Video" button because they should be able to play the video directly by clicking on it